python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -132,8 +138,8 @@ def asset(
             contains letters, numbers, and _) and may not contain python reserved keywords.
         ins (Optional[Mapping[str, AssetIn]]): A dictionary that maps input names to information
             about the input.
-        non_argument_deps (Optional[Union[Set[AssetKey], Set[str]]]): Set of asset keys that are
-            upstream dependencies, but do not pass an input to the asset.
+        deps (Optional[Sequence[Union[AssetsDefinition, SourceAsset, AssetKey, str]]]):
+            The assets that are upstream dependencies, but do not pass an input value to the asset.
```

Comment:
I think that "input" is a little bit ambiguous. Maybe "but do not correspond to a parameter of the decorated function" would be more clear.